data/settings.js

```diff
@@ -4990,6 +5044,88 @@ It can include key = value pairs for assigning variables the desired value
 upon Dovecot startup.`
 	},
 
+	inet_listener: {
+		tags: [ 'service' ],
+		values: setting_types.NAMED_LIST_FILTER,
+		seealso: [ 'service', 'inet_listener_name' ],
+		text: `
+Creates a new network socket listener for a [[setting,service]]. The filter
+name refers to the [[setting,inet_listener_name]] setting.`
+	},
+
+	inet_listener_name: {
+		tags: [ 'service' ],
+		values: setting_types.STRING,
+		seealso: [ 'inet_listener_type', 'inet_listener' ],
+		text: `
+Name of this listener. It is meant to be descriptive for humans (e.g. \`imap\`,
+\`imaps\`), but it is also used by services to determine the socket type,
+unless [[setting,inet_listener_type]] overrides it. The
+[[setting,inet_listener]] filter name refers to this setting.`
+	},
+
+	inet_listener_type: {
+		tags: [ 'service' ],
+		added: {
+			settings_inet_listener_type_added: false,
+		},
+		values: setting_types.STRING,
+		text: `
+Listener type. This string value has service-specific meaning and is used to
+distinguish different listener types that one service may employ.`
+	},
+
+	inet_listener_address: {
+		tags: [ 'service' ],
+		values: setting_types.IPADDR,
+		text: `
+Overrides the [[setting,listen]] setting for this listener.`
+	},
+
+	inet_listener_port: {
+		tags: [ 'service' ],
+		values: setting_types.UINT,
+		default: 0,
+		text: `
+Port number where to listen. \`0\` disables the listener.
+
+::: warning
+Although this is an unsigned integer. Only values between \`0\` and \`65535\`
+are supported.
```

Comment:
Also complained a bit about this to Michael's intercept merge request that we really should have setting_types.IN_PORT like in the code there is already. Then you wouldn't need to sprinkle around these kind of warnings.

Also, it should be a single sentence. Probably with comma: `integer, only`

Reply:
I added a new type and replaced the current values for all available port settings as of this MR.

Reply:
I don't see any such changes. Did you push them?

Reply:
Hmm no, I'm just having trouble with github

Reply:
No, not yet. I will keep an eye out for these beginning now.